### ISSUE TITLE
Still stick with `WARNNING: Audio Player: ao_open_live() failed: Cannot open the device: alsa` on arch linux (2021-09-13)

### ISSUE THREAD

**Issue by zw963** [goldendict/goldendict#1414]
![image](https://user-images.githubusercontent.com/549126/133029661-5f842f1e-3417-4eb9-b668-0853e0e9ba3a.png)

Is there a way to debug this issue?

I use latest Arch linux, this issue both happen on my two laptop. (both arch)

```sh
 ╰─ $ pacman -Q |grep golden
goldendict-qt5-git 1.5.0.RC2.490.gbaff02a1-1
```

following is some log when i run goldendict directly from konsole and pronunce failed.

```
getResource: gdlookup://localhost?word=hello&group=1
scheme: gdlookup
host: localhost
>>>Sugg:  st:hello
====reading 16384 bytes
====reading 16384 bytes
====reading 16384 bytes
====reading 16384 bytes
====reading 16384 bytes
some body finished
one not finished.
offset = 4f2e728
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
====reading 16384 bytes
====reading 16384 bytes
====reading 16384 bytes
====reading 16384 bytes
clicked QUrl("gdau://d292972ee4d31595f0ae6c3b45528c0b/media/english/ameProns/hello.mp3")
getResource: gdau://d292972ee4d31595f0ae6c3b45528c0b/media/english/ameProns/hello.mp3
scheme: gdau
host: d292972ee4d31595f0ae6c3b45528c0b
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
Codec open: MP3 (MPEG audio layer 3): channels: 1, rate: 22050, format: fltp
ao_open_live(): Advanced Linux Sound Architecture (ALSA) output: channels: 1, rate: 22050, bits: 16
```

this package support two driver,  if change to Qt multimedia, it works!

![image](https://user-images.githubusercontent.com/549126/133029492-4c29ec72-9018-4ea1-92cf-4c8d5ad10791.png)

**Comment by vedgy** [comment]
Duplicate of #710?

**Comment by zw963** [comment]
Yes, i check #710 too, change /etc/libao.conf not works for me when i use arch linux official package. (pacman -S goldendict)<goldendict-1.5.0RC2-9>

![image](https://user-images.githubusercontent.com/549126/133193791-8fcc8c3f-2f9e-4822-8767-7477f8ab81be.png)

but, after i change to use git version. (pacman -S goldendict-qt5-git) <goldendict-qt5-git-1.5.0.RC2.490.gbaff02a1-1>,  it works when use FFmpeg+libao if change /etc/libao.conf config.

So, if we should release a new version  for this bugfix?

**Comment by vedgy** [comment]
> So, if we should release a new version for this bugfix?

That's up to GoldenDict's maintainers, and they haven't tagged or released new versions in more than 5 years for some reason. See e.g. #839.

Please close this issue because it is fixed in *master* by #1273. The *libao.conf* adjustment is a system configuration matter, not a bug.
